src/cli.ts

```diff
@@ -535,5 +535,13 @@ program
   .action(() => {
     // eslint-disable-next-line @typescript-eslint/no-var-requires
     const pjson = require('../package.json');
+    const pythonRequirements = fs
```

Comment:
why aren't we using `starknet --vesion`? instead we're looking for an entry in `requirements.txt`

Reply:
Good point. Will update